parsing/location.mli

```diff
@@ -104,9 +107,6 @@ type error =
 
 exception Error of error
 
-val print_error_prefix: formatter -> unit -> unit
```

Comment:
I'm not against removing this function from the API, but is this related to the current proposal?